public/requests.js

```diff
@@ -0,0 +1,122 @@
+import md5_encode from './MD5.js'
+
+/**
+ *  добавление eventListener на кнопку поиска трека
+ */
+const search_input = document.querySelector('.header__search');
+search_input.addEventListener('keydown', function(e) {
+    if (e.keyCode === 13) {
+        send_request('GET',{method:'track.search',param:`&track=${search_input.value}`,track:search_input.value}).then(
+            res => {
+                if (res.hasOwnProperty('results')) {
+                    get_content('search',res.results.trackmatches.track)
+                }
+                else {
+                    alert('Произошла ошибка поиска, повторите попытку')
+                }
+            }
+        )
+    }
+});
+
+/**
+ *  добавление eventListener на кнопку любимых треков
+ */
+const fav_button = document.querySelector('.fav');
+fav_button.addEventListener('click', function() {
+    send_request('GET',{method:'user.getlovedtracks',param:'&user=Foxyb0y'}).then(
+        res => get_content('favorites',res.lovedtracks.track)
+    )
+});
+
+/**
+ *
+ * @param method метод Апи (GET или POST)
+ * @param params
+ * @returns {Promise} Выполненный промис
+ */
+async function send_request(method,params) {
+    const url = 'http://ws.audioscrobbler.com/2.0/';
```

Comment:
лучше слать запросы по https
```suggestion
    const url = 'https://ws.audioscrobbler.com/2.0/';
```